src/server.c

```diff
@@ -3241,10 +3241,18 @@ void call(client *c, int flags) {
     if (server.fixed_time_expire++ == 0) {
         updateCachedTimeWithUs(0,call_timer);
     }
+    #if defined(USE_PROCESSOR_CLOCK)
```

Comment:
A compile time check is not enough. There's also a runtime fallback, see monotonicInit()
I guess we need some infrastructure to support runtime conditions. Maybe it's also a good idea to expose this info in the INFO command..